src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,64 @@
 package mate.jdbc;
 
+import java.util.Optional;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        final ManufacturerDao manufacturerDao = (ManufacturerDao) injector
```

Comment:
You need to use service against dao in Main. 
This point was in the common mistakes, please check all your Main and change all Dao to Services
And also add ManufacturerDao to your github project when you next time will push it.